pounders/py/pounders.py

```diff
@@ -73,11 +75,16 @@ def pounders(fun, X0, n, npmax, nfmax, gtol, delta, nfs, m, F0, xkin, L, U, prin
     xkin    [int] Index of point in X representing approximate minimizer
     """
     if hfun is None:
-        hfun = lambda F: np.sum(F**2)
+
+        def hfun(F):
+            return np.sum(F**2)
+
         from .general_h_funs import leastsquares as combinemodels
 
     # choose your spsolver
     if spsolver == 2:
+        _FILE_LOCATION = Path(__file__).resolve().parent
+        sys.path.append(os.path.join(_FILE_LOCATION, "../../minq/py/minq5/"))
```

Comment:
I see you imported `Path`.  Might want to use it instead of `os.path` now for the path management.  Split `"../../minq/py/minq5/"` into individual arguments to avoid using `/`.

Does this work when you use the installed python package?  Since `minq` isn't in the package, I would guess that the relative path hardcoded here will not work.

Insist that user must set the python path appropriately?  Have them set a `minq` env var that the package can use to set the python path internally?

Reply:
I agree. We now try the import and if the module isn't found, we point the user in the right location.